css/sponsor.css

```diff
@@ -1,41 +1,105 @@
+* {
```

Comment:
you should probably just put border-box on the stuff you need to have border-box on and stay away from `* { ... }` style css declarations in general. something like `body { ... }` would suffice